frontstage/views/sign_in/errors_render.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from abc import abstractmethod
+
+from typing import Dict
+
+from flask import render_template
+from structlog import wrap_logger
+
+from frontstage.notifications.notifications import AlertViaGovNotify
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class RenderValidation:
+    """
+    This is the basic class that identify a generic routing type for any validation error that needs to be re routed.
+    All the abstract methods are mandatory to implement
+    """
+
+    def __init__(self, page: str, data: Dict):
+        """
+        :type page: page destination for the route function
+        :type data: it s dictionary of values to pass to the route function
+        """
+        self.page = page
+        self.data = data
+
+    @abstractmethod
+    def log_message(self, **log_parameters) -> str:
+        pass
+
+    def notify_user(self, *parameters):
+        return self
+
+    def route_me(self, form):
+        """
+        All the sub class will implement this method for default behavior.
+        :param form: is passed as argument from the flask app
+        :param params: is any optinal values that needs to be passed to the render template.
+        :return: render template
+        """
+        return render_template(self.page, form=form, data=self.data)
+
+
+class RenderAccountLocked(RenderValidation):
+
+    def log_message(self, *log_parameters):
+        logger.info('user account locked for too many attempts. UAA server error message : {}'.format(log_parameters))
```

Comment:
For consistency sake can this be capitalised first word like the rest of the logs

Reply:
I can see we are focus on important things . Changed.